features/mbedtls/targets/TARGET_Samsung/mbedtls_device.h

```diff
@@ -0,0 +1,39 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2015-2016 Nuvoton
```

Comment:
add SDPX identifiers to new files please

Reply:
https://github.com/ARMmbed/mbed-os/pull/12106/files#diff-05fa2e6d0809550427518de9236f75b4R19